File: readme/cn/selfhost/SDK/webfont-SDK-PHP(for1.22.2)/source/model/FontFaceParam.php
<?php
/**
 * Created by PhpStorm.
 * User: niyonghang
 * Date: 2020/2/27
 * Time: 3:16 PM
 */

namespace source\model;


class FontFaceParam
{
    private $accessKey;
    private $content;
    private $tag;

    // 类似：a/b/font1
    private $url;

    /**
     * FontFaceParam constructor.
     * @param $accessKey
     * @param $content
     * @param $tag
     * @param $url
     */
    public function __construct($accessKey, $content, $tag, $url)
    {
        $this->accessKey = $accessKey;
        $this->content = $content;
        $this->tag = $tag;
        $this->url = $url;
    }


    /**
     * @return mixed
     */
    public function getAccessKey()
    {
        return $this->accessKey;
    }

    /**
     * @param mixed $accessKey
     */
    public function setAccessKey($accessKey)
    {
        $this->accessKey = $accessKey;
    }

    /**
     * @return mixed
     */
    public function getContent()
    {
        return $this->content;
    }

    /**
     * @param mixed $content
     */
    public function setContent($content)
    {
        $this->content = $content;
    }

    /**
     * @return mixed
     */
    public function getTag()
    {
        return $this->tag;
    }

    /**
     * @param mixed $tag
     */
    public function setTag($tag)
    {
        $this->tag = $tag;
    }

    /**
     * @return mixed
     */
    public function getUrl()
    {
        return $this->url;
    }

    /**
     * @param mixed $url
     */
    public function setUrl($url)
    {
        $this->url = $url;
    }


}

File: readme/cn/selfhost/SDK/webfont-SDK-java(for1.22.2)/cn/webfont/client/model/FontFaceParam.java
package cn.webfont.client.model;

import lombok.Data;

@Data
public class FontFaceParam {
  private String accessKey;
  private String content;
  private String tag;
  // 类似：a/b/font1
  private String url;

  public FontFaceParam(String accessKey, String content) {
    this(accessKey, content, "", "");
  }

  public FontFaceParam(String accessKey, String content, String tag) {
    this(accessKey, content, tag, "");
  }

  public FontFaceParam(String accessKey, String content, String tag, String url) {
    this.accessKey = accessKey;
    this.content = content;
    this.tag = tag;
    this.url = url;
  }
}
